commands/command_output.py

```diff
@@ -0,0 +1,8 @@
+from commands.status import Status
+
+class CommandOutput:
+    def __init__(self, command, result=None, exception=None):
+        self.result = result
+        self.status = Status.SUCCESS.value if exception is None else exception.status.value
```

Comment:
Self.status = exception.status.name if it isnot none

the exception parent class  has a message. When you throw the exception, you should pass a message in the constructor

Reply:
Then you get the message here